### ISSUE TITLE
Feature Request: Add option to view PGN files

### ISSUE THREAD

**Issue by LichessUser** [veloce/lichobile#1403]
Hello, I just downloaded some PGN files to enjoy annotated games and learn from them. But I've to download a PGN viewer in order to open the file which I don't like because most of the apps show advertisement and annoying pop-ups. I would love to have a PGN viewer on the lichess app so that I don't have to look for another apps. Try to Insert this feature into the app. Lichess app is great and I love it :).(Sorry for bad English)
Thank you!

**Comment by Tiberian64** [comment]
I don't think you need a separate viewer.  You can import your PGN into a Study in Lichess.  Go to Learn -->  Study --> Create a New Study.  In the Study, Create a New Chapter.  In the dialogue box you can choose Empty (no import), Editor, URL, FEN, or PGN.  Choose PGN and paste your PGN.  It's a great way to review games from different sites you play on where you can export your game as a PGN.
